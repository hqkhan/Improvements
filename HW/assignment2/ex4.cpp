/* ECE373
 * Filename: ex4.cpp
 *
 * Problem: Make a while loop that loops until the user types y. Then make a
 *  for loop that will count from 1 to 5.
 */
#include <iostream> 

using namespace std;

int main()
{
	string inputVal;
	int ii;

	cout << "Do you want to continue?:";
	cin >> inputVal;

	/* http://www.cplusplus.com/reference/string/string/
	* inputVal is a String, so you can't do a direct conditional
	* statement with stringVar = stringVar2.
	* The above gives a reference for some of the String functions
	* in the String class.
	*/

	/*Modify to add a while loop*/
	while(true){
		if (inputVal.compare("y") != 0) {
			cout << "Do you want to continue?:";
			cin >> inputVal;
		}
		else break;
	}

	/*Modify to add a for loop*/
		
		for(ii = 1;ii<=5;ii++) cout << ii << endl;
	
}

