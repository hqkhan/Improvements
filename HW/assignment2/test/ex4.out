Do you want to continue?:Do you want to continue?:Do you want to continue?:1
2
3
4
5
